cumulus/polkadot-parachain/src/command.rs

```diff
@@ -86,27 +83,21 @@ fn runtime(id: &str) -> Runtime {
 		Runtime::Shell
 	} else if id.starts_with("seedling") {
 		Runtime::Seedling
-	} else if id.starts_with("asset-hub-polkadot") | id.starts_with("statemint") {
-		Runtime::AssetHubPolkadot
-	} else if id.starts_with("asset-hub-kusama") | id.starts_with("statemine") {
-		Runtime::AssetHubKusama
```

Comment:
Same, throughout this file. The node needs to run `asset-hub-polkadot`.

Reply:
This is one of the issues that I couldn't work out originally that I mentioned previously, from talking to nacho I understood that people running `asset-hub-(polkadot|kusama)` would use a wasm blob, but maybe I misunderstood.

If this this the case then we have issues because `Runtime::(AssetHubPolkadot|AssetHubKusama)` depend on having a copy of the runtimes somewhere.

Reply:
Yes it doesn't run a native runtime (uses the Wasm blob), but it needs things in the chain spec to know which bootnodes to connect to to fetch the Wasm blob.

Reply:
@bkontur this was the conversation about keeping these.